### ISSUE TITLE
Reverse playback queue

### ISSUE THREAD

**Issue by simon-auch** [TeamNewPipe/NewPipe#3815]
#### Describe the feature you want

It would be nice if the order of currently playing videos could be reversed.
This would be very handy, since when using the `play all` button videos are played back from newest to oldest (I think). This behavior however is not wanted in some situations (eg. podcasts (with a story line)).

#### How will you/everyone benefit from this feature?

In many cases I want to watch videos in the order they were released (basically everything that has a storyline over multiple videos, be it games, podcasts, etc).
Rearranging hundreds of videos by hand or playing each video by hand isn't really fun, so I think this would be a nice addition.

**Comment by eladkarako** [comment]
I join-in this weird request,  
a few of my favourite channels such as have a playlist ordered "most recent on top", 
but sometimes it makes more sense to play such playlists:  https://www.youtube.com/watch?v=stYYz2bH9FY&list=PLNo3_vmBGawcePEtYKDVXhTe_Q9Bymf9m&disable_polymer=1 from a logical point of view, which is from bottom to up. 

For the lack of support, I probably should create a new playlist with the same videos in a different order...

Would like that feature!

**Comment by pawlosck** [comment]
Yes, I agree. It's great feature. I have some channels which contains a lot of short videos and I would like to watch whole channel from beginning to latest video.

**Comment by joshatt** [comment]
+1.
This will be wonderful for audiobook, podcast or lectures. 
And, "sorting playlist" is not enough, audiobook podcast or lectures are not limited to playlist format, it'll be better if channels are also covered. So, either channel or playlist,  when new video added, a simple "play in reverse order" button or option will cover all situations.

**Comment by SameenAhnaf** [comment]
@zorbathegreek #5741

**Comment by desbest** [comment]
[Pipepipe](github.com/infinityloop1308/pipepipe), a fork of newpipe, has this feature.
